.github/actions/setup-build/action.yml

```diff
@@ -63,10 +63,15 @@ runs:
         fi
       shell: bash
 
+    - name: Disable -dynamic
```

Comment:
I still think this needs a comment.

Reply:
there is one in the .cabal file, but maybe could be refer it here

Reply:
finally added a full comment with ci specific caveats